how_to_add_new_hotspot.md

```diff
@@ -0,0 +1,142 @@
+# How To Add A New Hotspot Into Nebra Firmware
+
+The Nebra firmware stack has been designed to support various different architectures and vendors. Hence it is very flexible by design. But this also gives the developers a responsibility about supplying correct hardware configuration.
+
+This configuration information has to be supplied in different places of the stack. Let's review them one by one.
+
+## Checklist for adding a new hotspot type
+
+* Add variant into `variant_definitions` in hm-pyhelper repository
+* Create custom inputs (e.g. additional buttons) and outputs (e.g. additional LED's) in the new variant and update the logic in other repositories if necessary.
+* Release and publish it via CI/CD pipeline
+* Create related production fleets as "helium-<freq>-<device_type>"
+* Edit configuration parameters of the fleet
+* Add `VARIANT` and `FREQ` environment variables into fleet variables
+* Update automation steps in this repository for newly created fleets
+* Add sentry environment variables into fleet variables
```

Comment:
we will also need to add:
HELIUM_MINER_HEIGHT_URL to the production and testnet fleets
OVERRIDE_CONFIG_URL to the testnet fleets

Reply:
Done.